modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -58,6 +64,52 @@ public String getFilesystemFriendlyName() {
                 assertEquals("Recorded file not found", 1, files.length);
             }
         }
+
+        @Test
+        public void recordingTestThatShouldHaveCorrectDuration() throws IOException {
+            final String flvFileTitle = "ChromeThatRecordsAllTests-recordingTestThatShouldBeRecordedAndRetained";
+            final String flvFileNameRegEx = "PASSED-" + flvFileTitle + ".*\\.flv";
+
+            try (
+                BrowserWebDriverContainer chrome = new BrowserWebDriverContainer()
+                    .withCapabilities(new ChromeOptions())
+                    .withRecordingMode(RECORD_ALL, vncRecordingDirectory.getRoot())
+                    .withRecordingFileFactory(new DefaultRecordingFileFactory())
+            ) {
+                chrome.start();
+
+                doSimpleExplore(chrome);
+                chrome.afterTest(new TestDescription() {
+                    @Override
+                    public String getTestId() {
+                        return getFilesystemFriendlyName();
+                    }
+
+                    @Override
+                    public String getFilesystemFriendlyName() {
+                        return flvFileTitle;
+                    }
+                }, Optional.empty());
+
+                String recordedFile = vncRecordingDirectory.getRoot().listFiles(new PatternFilenameFilter(flvFileNameRegEx))[0].getCanonicalPath();
+
+                ToStringConsumer dockerLogConsumer = new ToStringConsumer();
+
+                try( GenericContainer container = new GenericContainer<>("jrottenberg/ffmpeg:3.2-alpine38") ) {
```

Comment:
In general, we like to avoid using 3rd party images as part of our CI (if we can). I assume we could use the updated `vnc-recorder` container here as well, or?

Reply:
Done.